### PR TITLE
Sorry CI

### PR DESCRIPTION
**Pull request by Geevies**
i'm smart i promise

### CODE REVIEW

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

File: code/modules/mob/living/carbon/human/inventory.dm [hunk 1]
```diff
@@ -339,7 +339,8 @@ This saves us from having to call add_fingerprint() any time something is put in
 		if(gear["slot"] == slot)
 			W.screen_loc = gear["loc"]
 			break
-	hud_used.hidden_inventory_update()
+	if(hud_used) // geeves broke the CI on 22/03/2021
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/inventory.dm:342]:
```suggestion
	if(hud_used)
```
no spraypaint

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/inventory.dm (hunk 1 above)
Comment (MarinaGryphon) [inline code/modules/mob/living/carbon/human/inventory.dm:342]:
```suggestion
```

File: code/modules/mob/living/carbon/human/inventory.dm [hunk 2]
```diff
@@ -339,7 +339,8 @@ This saves us from having to call add_fingerprint() any time something is put in
...
+		hud_used.hidden_inventory_update()
```
Comment (MarinaGryphon) [inline code/modules/mob/living/carbon/human/inventory.dm:343]:
```suggestion
	hud_used?.hidden_inventory_update()
```
This is easily onelined.

**Review by MarinaGryphon — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/inventory.dm (hunk 1 above)
Comment (Geevies) [reply code/modules/mob/living/carbon/human/inventory.dm:342]: why not

File: code/modules/mob/living/carbon/human/inventory.dm (hunk 2 above)
Comment (Geevies) [reply code/modules/mob/living/carbon/human/inventory.dm:343]: that's disgusting

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**